amdctl.c

```diff
@@ -623,12 +676,17 @@ float getCpuMultiplier(const int CpuFid, const int CpuDid) {
 		case AMD17H:
 		case AMD19H:
 			return (CpuFid * VID_DIVIDOR1) / (CpuDid * VID_DIVIDOR2);
+                case AMD14H:
+                        return 0;
```

Comment:
Can this be calculated maybe using the same math as the clockspeed but dividing by the REFCLK?

Reply:
I suppose it could, but since I wasn't 100% sure (the docs only talk about a divider, never about a multiplier) I just left it at 0. 

Logically it would make sense to take the clockspeed and divide it by the reference clock, as you pointed out.

Reply:
Sounds good, will make those changes locally, thanks!